frontend/ui/widget/touchmenu.lua

```diff
@@ -381,7 +381,6 @@ function TouchMenu:init()
         -- pad with 10 pixel to align with the up arrow in footer
         HorizontalSpan:new{width = 10},
         LineWidget:new{
-            style = "dashed",
             dimen = Geom:new{
                 w = self.item_width - 20,
                 h = 1,
```

Comment:
> It gets complicated because the line is too shallow using BB.COLOR_GREY at 300 DPI.
>
> So the gray level should be a function of the line thickness and the display DPI.

Is that complicated? Haven't tried but it seems like it should be pretty straightfoward:

```lua
w = self.item_width - Screen:scaleBySize(20),
h = Screen:scaleBySize(1),
```lua